style.css

```diff
@@ -80,30 +80,46 @@ nav {
   display: none;
 }
 
-.nav-menu {
+.blurred-bg {
   background: #6070ff;
   position: fixed;
+  top: 0.0000001px;
+  left: 0.0000001px;
+  width: 100%;
+  height: 100%;
+  opacity: 0.9;
+  filter: blur(5px);
+}
```

Comment:
- [ ] The blur shouldn't hide content of the site in desktop screen, I think you should consider that as well
![Screenshot from 2021-06-22 10-45-19](https://user-images.githubusercontent.com/56886064/122895442-675a2880-d348-11eb-8935-9a8f5bf541ba.png)

Reply:
I missed this problem. And fixed.